apps/rocko/src/commands/testing/confirm.ts

```diff
@@ -0,0 +1,49 @@
+import {
+	Button,
+	type ButtonInteraction,
+	ButtonStyle,
+	Command,
+	type CommandInteraction,
+	Row
+} from "@buape/carbon"
+
+export default class ConfirmCommand extends Command {
+	name = "confirm"
+	description = "Confirm a message"
+	async run(interaction: CommandInteraction) {
+		const done = await interaction.replyAndWaitForComponent({
+			content: "Confirm the message",
+			components: [
+				new Row([
+					new (class extends Button {
+						label = "Confirm"
+						style = ButtonStyle.Success
+						customId = "x-confirm"
+						async run(interaction: ButtonInteraction) {
+							return interaction.acknowledge()
```

Comment:
One thing I'm not sure about is this, the run() never actually gets called but it's still required...